### ISSUE TITLE
org-agenda incomprehensible behavior

### ISSUE THREAD

**Issue by s8v** [hlissner/doom-emacs#429]
Here are some things you should try before filing a bug report:

### Observed behavior

When I open agenda week view the following functions do not work:
- org-agenda-priority-down
- org-agenda-priority-up
- org-agenda-todo
- org-agenda-goto

### Expected behavior

I can change the priority of the task, the status of the task or go to the file.

### Steps to reproduce

[short-video](https://yadi.sk/i/FeV9_qMU3SinUr)
1. Open agenda without opening todo.org file.
2. Try to change priority, status, goto file.

### System information

<details>
<summary>Click to expand</summary>

```
Replace this line with the output of *one* of these commands:
DOOM Doctor
Running Emacs v25.3.1, commit d9d4b6afd51439f5789b8af19b41b5fe5a45124d

shell: /bin/zsh
Compiled with:
  JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB
  TOOLKIT_SCROLL_BARS NS MODULES
uname -a:
  Darwin MacBook-Air-Aleksandr.local 17.0.0 Darwin Kernel Version
  17.0.0: Thu Aug 24 21:48:19 PDT 2017;
  root:xnu-4570.1.46~2/RELEASE_X86_64 x86_64

Attempt to load DOOM: success! Loaded v2.0.9
Revision d9d4b6afd51439f5789b8af19b41b5fe5a45124d

----

test-emacs
test-windows
test-fonts
Found font material-design-icons.ttf
Found font weathericons.ttf
Found font octicons.ttf
Found font fontawesome.ttf
Found font file-icons.ttf
Found font all-the-icons.ttf
test-gnutls
test-tls
Validated https://elpa.gnu.org
Validated https://melpa.org
Successfully rejected https://self-signed.badssl.com
Successfully rejected https://wrong.host.badssl.com/
test-tar
Everything seems fine, happy Emacs’ing!
```
</details>

**Comment by AloisJanicek** [comment]
I have same experience.
If I have opened agenda file in buffer which has no focus and I have running clock in it, opening `org-agenda` results into question about if I want to kill buffer where is clocked task.
This isn't happening when buffer with agenda file is active (has focus).
So it looks like org-agenda kills agenda files shortly after it extracts agenda information.
I also tried to lock buffer containing agenda file (running `emacs-lock-mode` command on buffer, then bury it (`SPC z`) and then call org-agenda.
Message `Buffer "gtd.org" is locked and cannot be killed` also confirms my original assumption.  

(And maybe this is somehow connected to my experience with agenda sometimes having emacs keys active and sometimes evil one.)
